pkgs/fc/ceph/src/fc/ceph/lvm.py

```diff
@@ -40,8 +40,32 @@ class MdraidDevice(GenericBlockDevice):
     whole provided blockdevices without partitioning."""
 
     @classmethod
-    def create(cls, name: str, blockdevices: list[str]):
+    def create(
+        cls,
+        name: str,
+        # The mdraid individual member disks could also be generalised as
+        # GenericBlockDevices itself, but for simplicity directly use
+        # concrete string device paths here.
+        blockdevices: list[str],
+    ):
+        main_disks = blockdevices[:-1]
+        if main_disks:
```

Comment:
Invert condition flow, happy flow shall have no indentation but the guardian check has one

Reply:
Also do a concise check for the raid6 condition being fulfilled, raid6 needs at least 3 + the spare disk

`raid6_minimum_disks = 1 + PARITY + SPARE`